index.js

```diff
@@ -435,7 +435,7 @@ module.exports = (params) => {
 
   // Update the AWS http agent with the region
   if (typeof params.region !== false) {
```

Comment:
This should probably check for a `string` type, otherwise any type could be passed.